themes/default/content/docs/intro/console/organizations.md

```diff
@@ -15,6 +15,7 @@ aliases:
 ---
 
 Organizations are a space for you to collaborate on shared projects and stacks.
+New organizations have their own subscription.
```

Comment:
This doesn't feel quite right in this header section - might leave out here - but flesh out more below.